scripts/jenkins/ardana/ansible/roles/heat-generator/files/dns-update.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+import argparse
+
+import yaml
+
+
+def parse_commandline():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--dns-servers",
+        metavar="NAME",
+        help="A list of nameservers",
+        nargs="+",
+        default=[])
+    parser.add_argument(
+        "--ntp-servers",
+        metavar="NAME",
+        help="A list of ntp servers",
+        nargs="+",
+        default=[])
+    return parser.parse_args()
+
+
+if __name__ == "__main__":
+    options = parse_commandline()
+    print(options)
+
+    with open('cloudConfig.yml') as f:
+        data = yaml.load(f.read(), Loader=yaml.SafeLoader)
```

Comment:
can we leave this as an absolute path?  i.e. where is the file being opened.  Maybe add a new argument: the location of cloudConfig.yml as a full path, and add a default value to cover the simple case

Reply:
That's a good idea! I added a command line option.